x-pack/plugins/apm/public/components/app/correlations/log.tsx

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
Nit: Suggest to name this file `correlations_log.tsx` for consistency.

Reply:
Renamed here https://github.com/elastic/kibana/pull/109187/commits/6f9ffa172004850bd29b2eae35d4740b74aa040c